src/wfdirrd.c

```diff
@@ -1226,6 +1225,125 @@ DWORD DecodeReparsePoint(LPCWSTR szFullPath, LPWSTR szDest, DWORD cwcDest)
 	return reparseTag;
 }
 

```

Comment:
Why is the following code here instead of lfn.c with the other new code?

Reply:
Solved with [3d33bcd]